features/package-install.feature

```diff
@@ -1,5 +1,35 @@
 Feature: Install WP-CLI packages
 
+  Scenario: Install a package with an old package index url in package composer.json
+    Given an empty directory
+    And a composer.json file:
+      """
+      {
+        "repositories": {
+          "wp-cli": {
+            "type": "composer",
+            "url": "http://wp-cli.org/package-index/"
+          }
+        }
+      }
+      """
+    When I run `rm -rf /tmp/wp-cli-package-install-test/ && mkdir /tmp/wp-cli-package-install-test/ && mv composer.json /tmp/wp-cli-package-install-test/`
```

Comment:
If you place the `composer.json` in `package-directory` with ^, this step is unnecessary.